src/fontra/views/fontoverview/fontoverview.js

```diff
@@ -235,6 +328,104 @@ export class FontOverviewController extends ViewController {
   handleRemoteError(event) {
     //
   }
+
+  initContextMenuItems() {
+    // TODO: Implement the actions + how to handle them?
+    this.basicContextMenuItems.push({
+      actionIdentifier: "action.undo",
+    });
+    this.basicContextMenuItems.push({
+      actionIdentifier: "action.redo",
+    });
+  }
+
+  initActions() {
+    {
+      const topic = "0030-action-topics.menu.edit";
+
+      registerAction(
+        "action.undo",
+        {
+          topic,
+          sortIndex: 0,
+          defaultShortCuts: [{ baseKey: "z", commandKey: true, shiftKey: false }],
+        },
+        () => this.doUndoRedo(false),
+        () => this.canUndoRedo(false)
+      );
+
+      registerAction(
+        "action.redo",
+        {
+          topic,
+          defaultShortCuts: [{ baseKey: "z", commandKey: true, shiftKey: true }],
+        },
+        () => this.doUndoRedo(true),
+        () => this.canUndoRedo(true)
+      );
+    }
+    {
+      const topic = "0020-action-topics.menu.view";
+
+      registerAction(
+        "action.zoom-in",
+        {
+          topic,
+          titleKey: "zoom-in",
+          defaultShortCuts: [
+            { baseKey: "+", commandKey: true },
+            { baseKey: "=", commandKey: true },
+          ],
+          allowGlobalOverride: true,
+        },
+        () => this.zoomIn()
+      );
+
+      registerAction(
+        "action.zoom-out",
+        {
+          topic,
+          titleKey: "zoom-out",
+          defaultShortCuts: [{ baseKey: "-", commandKey: true }],
+          allowGlobalOverride: true,
+        },
+        () => this.zoomOut()
+      );
+    }
+    {
+      const topic = "0035-action-topics.menu.glyph";
+      registerAction("action.glyph.duplicate", { topic }, () => this.duplicateGlyph());
+      registerAction("action.glyph.delete", { topic }, () => this.deleteGlyphs());
+    }
+  }
+
+  async canUndoRedo(isRedo) {
+    // TODO: Do we really need this here? Or is it always true anyway?
+    return true;
+  }
+
+  async doUndoRedo(isRedo) {
+    // TODO: Implement the undo/redo functionality.
+    console.log(isRedo ? "redo" : "undo");
+  }
+
+  async zoomIn() {
+    console.log("font overview zoom in");
+  }
+
+  async zoomOut() {
+    console.log("font overview zoom out");
+  }
+
+  async duplicateGlyph() {
```

Comment:
Please do not invent new functionality. Do the basics first. Stubs for copy/paste/delete/undo etc. is fine. But this is just a random idea that isn't fully thought-through.

Reply:
I added all the stubs for discussion, because even the basics are not clear to me where they belong. Usually we have copy/paste/delete/undo under "Edit". But this is based on the the selected source in the editor view, am I right?
As we can switch between sources, would a "copy" in the menu bar "Edit" copy only the path of the source selected or the whole glyph with all sources? Where does Delete belongs "Edit" or "Glyph"? Would it be more like `action.delete-selection`, where we can delete multiple glyphs based on the 'Selection'? 
I removed `duplicateGlyph`, but did not add copy/paste or others. I think it would be good to discuss it first.

Reply:
I just commented "it should be populated with action identifiers" for the standard edit items, but I realize that doesn't give us menu titles if they're not defined. *Maybe* they should be: action idnetifier + default title.

There should be no functionality here at all. Views can register actions, and that will automatically enable the menu item appropriately.

Reply:
I added action identifiers for `getEditMenuItems` and `getViewMenuItems` as defaults. `getGlyphMenuItems` is empty. I don't know what this should contain as defaults. I keep this feedback still open, because it's still not full clear to me, where default items should belong to which menu.